src/GraphQL.Extensions.DI.Microsoft/ScopedFieldResolver.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using GraphQL.Resolvers;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace GraphQL.Extensions.DI.Microsoft
+{
+    public class ScopedFieldResolver<TReturnType> : FuncFieldResolver<TReturnType>
```

Comment:
Is `ScopedFieldResolver<TReturnType>` class used anywhere (`ScopedAsyncFieldResolver<TReturnType>` as well) ?

Reply:
Yes - ScopedFieldBuilderExtensions.cs

Reply:
I talked about `ScopedFieldResolver<TReturnType>` not `ScopedFieldResolver<TSourceType, TReturnType>`

Reply:
Oh. They might not be.